pkg/api/constants/consts.go

```diff
@@ -1,11 +1,14 @@
 package constants
 
 const (
-	ArtifactSpecRoutePrefix = "/oras/artifacts/v1"
-	RoutePrefix             = "/v2"
-	DistAPIVersion          = "Docker-Distribution-API-Version"
-	DistContentDigestKey    = "Docker-Content-Digest"
-	BlobUploadUUID          = "Blob-Upload-UUID"
-	DefaultMediaType        = "application/json"
-	BinaryMediaType         = "application/octet-stream"
+	ArtifactSpecRoutePrefix      = "/oras/artifacts/v1"
+	RoutePrefix                  = "/v2"
+	DistAPIVersion               = "Docker-Distribution-API-Version"
+	DistContentDigestKey         = "Docker-Content-Digest"
+	BlobUploadUUID               = "Blob-Upload-UUID"
+	DefaultMediaType             = "application/json"
+	BinaryMediaType              = "application/octet-stream"
+	SearchExtensionRoute         = "/v2/_search"
```

Comment:
Move extensions constants to extensions.go
SearchExtPrefix = RoutePrefix+'/_search'
ExtSearchPrefix <= follows dist-spec url scheme